docs/specifications/4-standard-compliance-suite.md

```diff
@@ -0,0 +1,166 @@
+# Standard Compliance Suite
+
+## Format
+In order to assert whether a new runtime implementation is correct, a formal
+compliance test suite is defined, that all language runtimes must fully
+implement before they can be deemed eligible for General Availability.
+
+This document describes these tests, as well as a general approach for ensuring
+conformance requirements are met in a systematic manner.
```

Comment:
I'm wondering if we should manage all this via markdown at all. 

Can we instead turn this into a JSON spec or something similar where there is a list of categories under which there is a list of test cases with path/name of the test file that implements it in that language? This can be the source of truth for compliance tests, i.e., if the test case is not in the spec file it's not part of the compliance test suite, it the test for a specific langunage is not in this spec file, that's a coverage gap.
We can then render it into a markdown table if you still like.

This may be over-engineering, I can't judge since I don't have the full picture; but my primary concern is that we (programmers and software engineers, in general) are terrible at maintaining documentation and keeping it consistent.

Reply:
I want to think of the spec as the source of truth - which is a golden master against which all implementations are assessed. Tools will be built and integrated as part of the continuous integration workflow to make sure consistency is there.

I don't want to have to reference all language implementation's tests in the spec, because this could not possibly include implementations made outside of this repository....